packages/mobile/src/redux/migrations.ts

```diff
@@ -362,4 +362,11 @@ export const migrations = {
       rampCashInButtonExpEnabled: false,
     },
   }),
+  27: (state: any) => ({
+    ...state,
+    app: {
+      ...state.app,
+      dappsExplorerEnabled: false,
```

Comment:
i was expecting to see `dappListApiUrl` here, maybe nice to add it for correctness? (i also notice `dappListApiUrl` was added to the v7 schema in the test schemas 🙈)

Reply:
Oh, it makes sense, I'll fix it.